docs/readme.txt

```diff
@@ -146,4 +146,4 @@ If you are looking for support, you can get it from
 
 Have fun!
 
-The wxWidgets Team, October 2015
+The wxWidgets Team, June 2016
```

Comment:
By the time of the next release, it probably will be... I just hope it will still be 2016 :-/